openlibrary/plugins/upstream/covers.py

```diff
@@ -9,7 +10,6 @@
 from openlibrary.plugins.upstream.models import Image
 from openlibrary.plugins.upstream.utils import get_coverstore_url, render_template
 
-from six.moves import urllib
 
 
```

Comment:
Three blank lines in a row is too much.  ;-)  Two please.

Reply:
Done 😄